pkg/spoke/controllers/manifestcontroller/manifestwork_controller.go

```diff
@@ -429,31 +537,34 @@ func buildManifestResourceMeta(index int, object runtime.Object, manifest workap
 		unstructuredObj := &unstructured.Unstructured{}
 		if err = unstructuredObj.UnmarshalJSON(manifest.Raw); err != nil {
 			errs = append(errs, err)
-			return resourceMeta, utilerrors.NewAggregate(errs)
+			return resourceMeta, gvr, utilerrors.NewAggregate(errs)
 		}
 		object = unstructuredObj
 	}
-	resourceMeta, err = buildResourceMeta(index, object, restMapper)
+	resourceMeta, gvr, err = buildResourceMeta(index, object, restMapper)
 	if err == nil {
-		return resourceMeta, nil
+		return resourceMeta, gvr, nil
 	}
 
-	return resourceMeta, utilerrors.NewAggregate(errs)
+	return resourceMeta, gvr, utilerrors.NewAggregate(errs)
 }
 
-func buildResourceMeta(index int, object runtime.Object, restMapper meta.RESTMapper) (resourceMeta workapiv1.ManifestResourceMeta, err error) {
+func buildResourceMeta(
+	index int,
+	object runtime.Object,
+	restMapper meta.RESTMapper) (resourceMeta workapiv1.ManifestResourceMeta, gvr schema.GroupVersionResource, err error) {
```

Comment:
past me should have caught this.  Can we eliminate the named return values for this method?